cmd/auth-handler.go

```diff
@@ -174,17 +222,37 @@ func checkRequestAuthType(ctx context.Context, r *http.Request, action policy.Ac
 		r.Body = ioutil.NopCloser(bytes.NewReader(payload))
 	}
 
-	if globalPolicySys.IsAllowed(policy.Args{
+	var claims = map[string]interface{}{}
+	s3Err = getClaimsFromToken(r, claims)
```

Comment:
We can avoid sending empty claims like this - it is not needed to verify the token signature.

Reply:
it is needed to extract the claims.